hikari/impl/special_endpoints.py

```diff
@@ -218,6 +220,19 @@ class GuildBuilder(special_endpoints.GuildBuilder):
     ] = attr.ib(metadata={attr_extensions.SKIP_DEEP_COPY: True})
     _roles: typing.MutableSequence[data_binding.JSONObject] = attr.ib(factory=list, init=False)
 
+    # Optional args that aren't kept hidden.
+    default_message_notifications: undefined.UndefinedOr[guilds.GuildMessageNotificationsLevel] = attr.ib(
+        default=undefined.UNDEFINED
+    )
+    explicit_content_filter_level: undefined.UndefinedOr[guilds.GuildExplicitContentFilterLevel] = attr.ib(
+        default=undefined.UNDEFINED
+    )
+    icon: undefined.UndefinedOr[files.Resourceish] = attr.ib(default=undefined.UNDEFINED)
+    region: undefined.UndefinedOr[voices.VoiceRegionish] = attr.ib(default=undefined.UNDEFINED)
+    verification_level: undefined.UndefinedOr[typing.Union[guilds.GuildVerificationLevel, int]] = attr.ib(
+        default=undefined.UNDEFINED
+    )
+
```

Comment:
These should go first

Reply:
Shouldn't they go after the required arguments rather than before?

Reply:
You fixed it already. I meant that they should go before the non-arguments